common/test/acceptance/tests/studio/base_studio_test.py

```diff
@@ -37,7 +37,7 @@ def populate_course_fixture(self, course_fixture):
         """
         pass
 
-    def log_in(self, user, is_staff=False):
+    def log_in(self, user, is_staff=False, no_login=False):
```

Comment:
This argument's presence is confusing. I'm still reading, so I'm sure you have a reason for it being here, but could you add something in the docstring explaining what it's for?